fabric-dimensions-v1/src/main/java/net/fabricmc/fabric/mixin/EntityHooks.java

```diff
@@ -0,0 +1,17 @@
+package net.fabricmc.fabric.mixin;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.Vec3d;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(Entity.class)
+public interface EntityHooks {
+    @Accessor
+    void setLastPortalDirectionVector(Vec3d vec);
```

Comment:
What exactly is this supposed to be for?

Reply:
This sets the vector that stores the `horizontalOffset` and `verticalOffset` variables that will be passed to `placeEntity`.